### PR TITLE
Set content-loaders limit with respect to the window-width

### LINKED ISSUE #3168: Content Loader in main page not fit for large screens
**Actual Behaviour**

The content loader in the main page is not fit for large screen sizes.

**Expected Behaviour**

It should adjust the size dynamically according to the screen size.

**Screenshots of the issue**

un

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #3168 

Changes: 
- Set limits to display loaders according to `window.innerWidth()`.

Demo Link: https://pr-3302-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
**After Changes**
Screenshot from 2020-01-17 14-57-05
Screenshot from 2020-01-17 14-57-18 (1)
Screenshot from 2020-01-19 10-21-23

### CODE REVIEW

**Comment by anishagg17** [comment]
Please do the same for table loaders too in my ratings and other  sections if required

**Comment by anishagg17** [comment]
please fix the width for blog , table loaders to if required.

**Comment by Dilshaad21** [comment]
@anishagg17 its only required in skills section as on other two sections it does not depend on window width.

File: src/components/cms/BrowseSkill/skillLoader.js [hunk 1]
```diff
@@ -14,9 +14,23 @@ const ContentContainer = styled.div`
   border: 0.5px solid #eaeded;
 `;
 
+const loaderLimit = () => {
+  let limit = 2;
+  if (window.innerWidth >= 1680) {
```
Comment (akshatnitd) [inline src/components/cms/BrowseSkill/skillLoader.js]: Store window.innerWidth in a variable and then use that variable.

Comment (Dilshaad21) [reply src/components/cms/BrowseSkill/skillLoader.js]: Sure :+1:

Comment (Dilshaad21) [reply src/components/cms/BrowseSkill/skillLoader.js]: It'll take some time

**Comment by Dilshaad21** [comment]
@akshatnitd now it's fine for all screen sizes

**Comment by Dilshaad21** [comment]
@akshatnitd it's working fine for me. I think surge link is not working.

**Pull request merged**